examples/rules_ios/iOSApp/Test/ObjCUnitTests/BUILD

```diff
@@ -15,7 +15,6 @@ rules_ios_ios_unit_test(
     name = "iOSAppObjCUnitTests",
     srcs = glob(["**/*.m"]),
     bundle_id = "rules-xcodeproj.example.objctests",
-    bundle_name = "iOS_App_ObjC_UnitTests",
```

Comment:
Custom bundle_name was broken 😞  https://github.com/bazel-ios/rules_ios/pull/754#issuecomment-1731796129

Reply:
I see you've got a [fix](https://github.com/bazel-ios/rules_ios/pull/770) up in `rules_ios` already, thx!

Reply:
So once that lands we can bring this back correct?

Reply:
Mind commenting this out and linking to the rules_ios PR or waiting until a new version is published so this isn't lost?

Reply:
+1 to trying to pick the fixed code here instead of commenting it out if possible

Reply:
Updated